config/main.py

```diff
@@ -3638,14 +3638,18 @@ def mtu(ctx, interface_name, interface_mtu, verbose):
     """Set interface mtu"""
     # Get the config_db connector
     config_db = ctx.obj['config_db']
-    if clicommon.get_interface_naming_mode() == "alias":
-        interface_name = interface_alias_to_name(config_db, interface_name)
-        if interface_name is None:
-            ctx.fail("'interface_name' is None!")
 
-    portchannel_member_table = config_db.get_table('PORTCHANNEL_MEMBER')
-    if interface_is_in_portchannel(portchannel_member_table, interface_name):
-        ctx.fail("'interface_name' is in portchannel!")
+    if not interface_name.startswith("PortChannel"):
+        # The checks are only for Ethernet interfaces and their aliases
+        if clicommon.get_interface_naming_mode() == "alias":
+            name_from_alias = interface_alias_to_name(config_db, interface_name)
+            if interface_name is None or name_from_alias == interface_name:
```

Comment:
With this added check you are requiring that once the device is configured with naming_mode alias it must use the alias name and not the SONiC interface name and it will cause this to fail?  Can we preserve the original design/check and not treat the case where the look up resulted to the same name as a failure scenario?